miner/worker.go

```diff
@@ -1055,6 +1059,10 @@ loop:
 			}
 			return atomic.LoadInt32(interrupt) == commitInterruptNewHead, circuitCapacityReached
 		}
+		// seal block early early if we're over time
+		if w.current.tcount > 0 && w.chainConfig.Clique != nil && uint64(time.Now().Unix()) > w.current.parentTimestamp+w.chainConfig.Clique.Period {
+			return false, true
```

Comment:
why not `return false, circuitCapacityReached`? this way we can reflect `circuitCapacityReached` more accurately?

Reply:
returning a constant `true`forces the caller to seal the block.

Reply:
Yes, that was my thinking. Skip subsequent invocations of `commitTransactions` inside `commitNewWork`.